docs/source/user_guide/pipelines.ipynb

```diff
@@ -125,7 +125,9 @@
    "metadata": {},
    "source": [
     "## Override Component Hyperparameter Ranges\n",
-    "To specify custom hyperparameter ranges, set the `custom_hyperparameters` property to be a dictionary where each key-value pair consists of a parameter name and range. AutoML will use this dictionary to override the hyperparameter ranges collected from each component in the component graph."
+    "To specify custom hyperparameter ranges, set the `custom_hyperparameters` property to be a dictionary where each key-value pair consists of a parameter name and range. AutoML will use this dictionary to override the hyperparameter ranges collected from each component in the component graph.\n",
+    "\n",
+    "These hyperparameter ranges can be passed in as lists or as `skopt.space.Categorical` values."
```

Comment:
I think we should clarify that only categorical spaces can be specified by either a list or `skopt.space.Categorical`? For other hyperparameters we use either `Real` or `Integer` and I don't think those can be specified by a list.